pkg/machine/applehv/config.go

```diff
@@ -0,0 +1,52 @@
+//go:build arm64 && !windows && !linux
+// +build arm64,!windows,!linux
```

Comment:
if this should only build  on macos then use the darwin build tag, !windows && !linux means it will try to build on freebsd for example.

Reply:
Needs to be adjusted in applehv/machine.go as well